release/models/optical-transport/openconfig-transport-types.yang

```diff
@@ -22,7 +22,13 @@ module openconfig-transport-types {
     "This module contains general type definitions and identities
     for optical transport models.";
 
-  oc-ext:openconfig-version "0.17.1";
+  oc-ext:openconfig-version "0.17.2";
```

Comment:
I think formally in [semver.org](https://semver.org/) this change is considered a minor rev vs. patch as we are adding data vs. bug fixing existing data.

Reply:
Done.